test/test_session_params.cpp

```diff
@@ -58,7 +58,7 @@ namespace
 
 	struct custom_plugin : plugin
 	{
-		void added(session_handle h) override
+		void added(session_handle& h) override
```

Comment:
I thought session_handle was essentially a shared_ptr<> to a session_impl. why is it passed by reference? if it shouldn't be passed by value, I imagine it should be at least const reference.